packages/api/src/bsky-agent.ts

```diff
@@ -0,0 +1,228 @@
+import { AtUri } from '@atproto/uri'
+import { AtpAgent } from './agent'
+import {
+  AppBskyFeedPost,
+  AppBskyActorProfile,
+  ComAtprotoRepoPutRecord,
+} from './client'
+
+export class BskyAgent extends AtpAgent {
+  get app() {
+    return this.api.app
+  }
+
+  getTimeline: typeof this.api.app.bsky.feed.getTimeline = (params, opts) =>
+    this.api.app.bsky.feed.getTimeline(params, opts)
+
+  getAuthorFeed: typeof this.api.app.bsky.feed.getAuthorFeed = (params, opts) =>
+    this.api.app.bsky.feed.getAuthorFeed(params, opts)
+
+  getPostThread: typeof this.api.app.bsky.feed.getPostThread = (params, opts) =>
+    this.api.app.bsky.feed.getPostThread(params, opts)
+
+  getPost: typeof this.api.app.bsky.feed.post.get = (params) =>
+    this.api.app.bsky.feed.post.get(params)
+
+  getLikes: typeof this.api.app.bsky.feed.getLikes = (params, opts) =>
+    this.api.app.bsky.feed.getLikes(params, opts)
```

Comment:
Definitely not too hung-up on it, but due to the use of the sugared `post.get()` I think that `getPost` may have unwrapped the result out of `response.data`, whereas `getLikes` and others return the whole `XRPCResponse` object.